riscv/src/compiler.rs

```diff
@@ -595,14 +590,35 @@ fn runtime() -> &'static str {
 fn process_statement(s: Statement) -> Vec<String> {
     match &s {
         Statement::Label(l) => vec![format!("{}::", escape_label(l))],
-        Statement::Directive(_, _) => panic!(""),
+        Statement::Directive(directive, args) => match (directive.as_str(), &args[..]) {
+            (
+                ".loc",
+                [Argument::Expression(Expression::Number(file)), Argument::Expression(Expression::Number(line)), Argument::Expression(Expression::Number(column)), ..],
+            ) => {
+                vec![format!("  debug loc {file} {line} {column};")]
+            }
+            (".file", _) => {
+                // We ignore ".file" directives because they have been extracted to the top.
+                vec![]
+            }
+            _ if directive.starts_with(".cfi_") => vec![],
```

Comment:
is this a TODO or won't do?

Reply:
It's a "let's ignore it for now". It's additional debugging information like function-local variables and stuff.